### ISSUE TITLE
Beestat not synching

### ISSUE THREAD

**Issue by Frankpc123** [beestat/app#354]
Beestat doesn’t seem to find or synch with my Eccobees. IPhone app and webpage both big working. 
Error:

Looks like you broke it again.
API call failed: Rate limit reached.
Sorry about that! This error has been logged and
will be investigated and appropriately punished.
Please reach out to contact@beestat.io if it
persists.
{
"success": false,
"data":
"error message":
"Rate limit reache
"error code": 1005

**Comment by ziebelje** [comment]
Hey! Did you reach out to me via email about this one? I can take a look if I haven't already; just need your thermostat serial or the registered email to look it up.

**Comment by Frankpc123** [comment]
Hi Jon,
It’s all good as of yesterday. I have a note to update the trouble log. Sorry about that. Interesting fix:
I tried Beestat via the webpage instead of my iPhone app. The Beestat webpage flipped to the Ecobee login page / two factor authentication. Once I did that both the Beestat webpage and Beestat iPhone app worked fine. 
I’m not sure if that was coincidence or not. I’m pretty sure I tried Beestat on the webpage last week and it didn’t synch and didn’t redirect to the Ecobee login. I think the Ecobee login expires after a certain time? Perhaps that is the issue? 

Thanks …. Really like Beestat ….

Frank
